submissions/lalalilulu/memory-game/script.js

```diff
@@ -0,0 +1,102 @@
+//start the game
+(function launchApp() {
+    const gameBoard = document.querySelector(".memory-game");
+    const fragment = document.createElement("div");
+    createImgArray().map(img => createCard(img)).forEach(card => fragment.appendChild(card));
+    gameBoard.innerHTML = fragment.innerHTML;
+})();
+
+function createImgArray() {
+    const imgArray = [];
+    for (let i = 1; i < 11; i++) {
+        imgArray.push("kitty-" + i);
+    }
+    imgArray.push(...imgArray);
+    return imgArray;
+}
+
+function createCard(image) {
```

Comment:
I'd recommend using template strings for generating the card. It will help you to simplify the render logic and get rid of the redundant functions